admin/admin-model.php

```diff
@@ -228,7 +228,12 @@ public function update_language( $args ) {
 		update_option( 'polylang', $this->options );
 
 		// And finally update the language itself
-		$description = maybe_serialize( array( 'locale' => $args['locale'], 'rtl' => (int) $args['rtl'], 'flag_code' => empty( $args['flag'] ) ? '' : $args['flag'] ) );
+		$description = maybe_serialize(
+			array_merge(
+				array( 'active' => $lang->active ? true : $lang->active ),
+				array( 'locale' => $args['locale'], 'rtl' => (int) $args['rtl'], 'flag_code' => empty( $args['flag'] ) ? '' : $args['flag'] )
```

Comment:
Since `locale` will became `locales` [here](https://github.com/polylang/polylang/pull/1192). Maybe we should make the change here right away.

Reply:
Another way to avoid future modification (if one day another feature is added in description) is to retrieve the term and its description, merge the modification and only then update the term?